src/BlockTypes/Breadcrumbs.php

```diff
@@ -34,12 +34,15 @@ protected function render( $attributes, $content, $block ) {
 			return;
 		}
 
-		$classname          = $attributes['className'] ?? '';
-		$classes_and_styles = StyleAttributesUtils::get_classes_and_styles_by_attributes( $attributes );
+		$classname             = $attributes['className'] ?? '';
+		$classes_and_styles    = StyleAttributesUtils::get_classes_and_styles_by_attributes( $attributes );
```

Comment:
`align` and `text_align` are missing from `StyleAttributesUtils::get_classes_and_styles_by_attributes()`.

Reply:
> I think instead of using get_align_class_and_style, we should add align support to StyleAttributesUtils::get_classes_and_styles_by_attributes() instead
>
> align and text_align are missing from StyleAttributesUtils::get_classes_and_styles_by_attributes().
I like the idea. It's covered in [this commit](https://github.com/woocommerce/woocommerce-blocks/pull/8248/commits/cd71a6afa8c42f6c5a55eaa4e10f442174cec23a).

> $attributes['className'] should also be included but let's work on it in a separate refactoring PR.
I agree that deserves a separate PR, since it would be more impactful change (requires adjustments in nearly every block and it would bloat this PR). I'll create an issue 👍